torch_geometric/nn/conv/message_passing.py

```diff
@@ -444,7 +444,7 @@ def update(self, inputs: Tensor) -> Tensor:
         """
         return inputs
 
-    def edge_update(self) -> Tensor:
+    def edge_update(self, x_j: Tensor) -> Tensor:
```

Comment:
I guess this needs to be added for TorchScript to work? I would prefer to leave it as it is.

Reply:
Right. I'll follow up on this comment in your below comment on this. Possible to avoid it.